### DISCUSSION TITLE
Send notifications after updating forms

### DISCUSSION THREAD

**Discussion by kobeaerts** [verbb/formie#1673] [Q&A] [answered]
Is it possible to send a notification after a certain field has been updated?
My use case is the following:

1. I have a form that gets saved normally, this form contains fields for the payment status (empty, unpaid, paid)
2. In the after save event of those forms I send the user to a Stripe checkout
3. After the checkout I update the form payment status
4. Only if that payment_status field is equal to "paid" the notification should be send

Or another option: is it possible to programatically send a notification in my Controllers?

**Comment by engram-design** [comment, marked as answer]
It's not possible to do this at the moment, sorry.

You can certainly trigger a notification to be sent on-demand:

```php
$notification = Formie::$plugin->getNotifications()->getNotificationById(123);
$submission = Submission::find()->id(123)->one();
Formie::$plugin->getSubmissions()->sendNotificationEmail($notification, $submission);
```

See https://github.com/verbb/formie/blob/9925889a29edae6dbb775ebd9def359ad05f67c6/src/controllers/SubmissionsController.php#L856

**Reply by kobeaerts** [reply]
And how do I get the notifications of a certain form 😂
I tried looking for it but there doesn't seem to be a function on the Form model?

**Reply by kobeaerts** [reply]
Nevermind, found it ;)

**Reply by engram-design** [reply]
It'll be [`$form->getNotifications()`](https://github.com/verbb/formie/blob/9925889a29edae6dbb775ebd9def359ad05f67c6/src/elements/Form.php#L1252) for future reference
